api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -259,6 +266,32 @@ public void removeUserFromBillingProject(String email, String projectName) {
         });
   }
 
+  @Override
+  public void addOwnerToBillingProject(String ownerEmail, String projectName) {
+    addRoleToBillingProject(ownerEmail, projectName, OWNER_FC_ROLE);
+  }
+
+  @Override
+  public void removeOwnerFromBillingProject(
+      String ownerEmailToRemove, GoogleCredential retainingOwnerCredential, String projectName)
+      throws IOException {
+
+    retainingOwnerCredential.refreshToken();
```

Comment:
nit: Maybe pass in the access token instead of the GoogleCredential so we can push all of the credentials logic into the caller versus having it split between the caller and this function.